verification/src/http_server/handlers/verification/solidity/types.rs

```diff
@@ -64,12 +64,15 @@ impl TryFrom<MultiPartFiles> for CompilerInput {
 
 #[derive(Clone, Debug, Deserialize)]
 pub struct StandardJson {
-    input: CompilerInput,
+    input: String,
 }
 
-impl From<StandardJson> for CompilerInput {
-    fn from(input: StandardJson) -> Self {
-        input.input
+impl TryFrom<StandardJson> for CompilerInput {
+    type Error = anyhow::Error;
+
+    fn try_from(input: StandardJson) -> Result<Self, Self::Error> {
+        serde_json::from_str(&input.input)
+            .map_err(|e| anyhow::Error::msg(format!("cannot parse input: {}", e)))
```

Comment:
Let's maybe provide more descriptive name? I suggest something like "content is not valid standard json: {}"